runtime/tests/common/mock.rs

```diff
@@ -74,6 +57,10 @@ impl ExtBuilder {
 		self.initial_balance = initial_balance;
 		self
 	}
+	pub fn initial_authorities(mut self, initial_authorities: &[AuthorityKeys]) -> Self {
```

Comment:
set_initial_authorities seems to be a better name

Reply:
It seems to be convention for the rust [builder pattern](https://doc.rust-lang.org/1.0.0/style/ownership/builders.html) to name setters like this so will leave it as is for consistency